addons/gitlab/tests/utils.py

```diff
@@ -1,5 +1,8 @@
 import mock
```

Comment:
This is a `gitlab` file, not `github`. It probably doesn't need to be changed.

Reply:
This is a re-boot of Corbin's old PR and I believe he wrote this to consolidate the mocking, but I can remove it if you want, seems harmless though.